### DISCUSSION TITLE
Web Password?

### DISCUSSION THREAD

**Discussion by florezriascos** [softwarecrash/Solar2MQTT-ESP8266#36] [Ideas]
Good afternoon gentlemen, I hope you are very well! I don't know much about code, that's why I appreciate your work which is fantastic... I wanted to propose that you include password to the configuration page and and to the configuration of MQTT or dashboard, since an intruder can copy and modify the desired values making this dangerous... Thank you very much gentlemen

**Comment by all-solutions** [comment]
Hello,
thank you very much for the suggestion.
One note: if an attacker is able to access the pages of the Wemos, then he is inside your network. I guess he has many other devices that are much more interesting. :wink:
You have not exposed the Wemos to the Internet?
